charts/graphistry-helm/templates/dask/dask-cuda-worker-daemonset.yaml

```diff
@@ -0,0 +1,141 @@
+{{ if eq .Values.dask.operator false }}
+{{ if eq .Values.global.devMode false }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: dask-cuda-worker
+  name: dask-cuda-worker
+spec:
+  replicas: {{ .Values.dask.workers }}  #needs to be num dask workers
+  selector:
+    matchLabels:
+      io.kompose.service: dask-cuda-worker
+  strategy:
+{{ if eq .Values.rollingUpdate true }}
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: {{.Values.maxSurge}}
+      maxUnavailable: "0%"
+{{ else }}
+    type: Recreate
+{{- end }}
+  template:
+    metadata:
+      labels:
+        io.kompose.network/grph: "true"
+        io.kompose.service: dask-cuda-worker
+    spec:
+      initContainers:
+      - name: "{{ lower .Chart.Name }}-init"
+        {{- if eq .Values.global.containerregistry.name "docker.io/graphistry" }}
+        image: "groundnuty/k8s-wait-for:latest" #DockerHub
+        {{ else }}
+        image: {{.Values.global.containerregistry.name}}/k8s-wait-for:latest
+        {{- end }}
+        resources:
+            {{- toYaml .Values.InitContainerResources | nindent 10 }}
+        imagePullPolicy: {{.Values.global.imagePullPolicy }} 
+        args: 
+        - "service"
+        - "nexus"
+      containers:
+        - args:
+            - dask-cuda-worker
+            - --interface
+            - eth0
+            - --dashboard-address localhost:8787
+            - --resources GPU=10
+            - dask-scheduler:8786
+          command:
+            - /entrypoints/rapids-entrypoint.sh
+          env:
```

Comment:
I think we can set `CUDA_VISIBLE_DEVICES` to 0 here, just to force the dask worker will use a single GPU, so we can manage the GPU assignation to the pods using kubernetes limits.

Reply:
will change in next release i have to test and make sure it doesnt block gpu, when i tried setting gpu requests at a per pod basis it was only connecting gpu to one pod at a time